frontend/src/components/Table/columns.tsx

```diff
@@ -0,0 +1,32 @@
+import { Button } from 'antd';
+import { IDataItem } from '../../types';
```

Comment:
not better with `/` in place of `../..` ? 

in `tsconfig.json`
`"compilerOptions": {
        "baseUrl": "./src", ...etc}`